covalent/cloud_resource_manager/core.py

```diff
@@ -27,13 +27,15 @@
 from typing import Any, Callable, Dict, List, Optional, Sequence, Union
 
 from covalent._shared_files.config import set_config
+from covalent._shared_files.defaults import get_default_sdk_config
 from covalent.executor import _executor_manager
 
 logger = logging.getLogger()
 logger.setLevel(logging.ERROR)
 handler = logging.StreamHandler(sys.stderr)
 logger.addHandler(handler)
 logger.propagate = False
+sdk_constants = get_default_sdk_config()
```

Comment:
We should use `get_config` instead of directly importing the defaults since they might've been changed by the user in the config file directly. Using `get_config` with specifically what is needed (instead of the entire sdk config) will take that into consideration as well.